src/gui/loginrequireddialog/loginrequireddialog.h

```diff
@@ -29,9 +29,17 @@ class LoginRequiredDialog : public QDialog
     Q_OBJECT
 
 public:
```

Comment:
For the class (I can't leave a review on line 27): is this the One And Only login dialog? If so, please document that.